str0m/Cargo.toml

```diff
@@ -9,6 +9,8 @@ tracing = "0.1"
 rand = "0.8"
 bitvec = "1"
 once_cell = "1"
+serde = "1"
+itertools = "0.10"
```

Comment:
Do we really need this? I don't want more dependencies than we need. Does it solve a big problem?

Reply:
I used it for group_by() alone. Up to you.

Reply:
Lets try without

Reply:
done

Reply:
done